cluster-provision/k8s/fetch-latest-cdi.sh

```diff
@@ -69,7 +69,13 @@ cdi_version="${tag_name/#v/}"
         curl -O -s -L -f "$file"
     done
 
-    while IFS= read -r -d '' file; do
-        mv "$file" "${file/#\.\/cdi-/\.\/cdi-$cdi_version-}"
-    done < <(find . -type f -name 'cdi*.yaml' -print0)
+    # Renaming files to cr.yaml and operator.yaml
```

Comment:
What is the need for this renaming? 

Can we not just take the updated files directly?

Reply:
we can, i just named them this way because the downloaded files are called cdi-cr, and cdi-operator
however cdi is implied

Reply:
its probably better just to take the manifests as they are released saves us having to rename etc.